components/VersionHistory/VersionHistory.vue

```diff
@@ -109,7 +114,8 @@ export default {
 <style lang="scss" scoped>
 .version-history-dialog {
   .version-history-container {
-    height: 290px;
+    height: 320px;
```

Comment:
`rem` or `em` units?

Reply:
Just updated these all to `em` check out my latest commit